assets/scripts/bundle.js

```diff
@@ -42244,6 +42244,10 @@ var AIRPORT_LOAD_LIST = exports.AIRPORT_LOAD_LIST = [{
     level: 'medium',
     name: 'San Francisco International Airport &#9983'
 }, {
+    icao: 'kpvd',
+    level: 'beginner',
+    name: 'Theodore Francis State Memorial Airport'
+}, {
```

Comment:
be sure to also add these lines to `https://github.com/zlsa/atc/blob/gh-pages/src/assets/scripts/airport/airportLoadList.js`